token-lending/program/src/processor.rs

```diff
@@ -2227,6 +2228,11 @@ fn get_switchboard_price(
     if *switchboard_feed_info.key == spl_token_lending::NULL_PUBKEY {
         return Err(LendingError::NullOracleConfig.into());
     }
+    if switchboard_feed_info.owner == &spl_token_lending::SWITCHBOARD_V2_MAINNET
+        || switchboard_feed_info.owner == &spl_token_lending::SWITCHBOARD_V2_DEVNET
```

Comment:
could someone spoof prices on mainnet by using the devnet private key (which is exposed)?